com.unity.render-pipelines.universal/Editor/Lighting/UniversalRenderPipelineLightUI.Drawers.cs

```diff
@@ -1,10 +1,25 @@
 namespace UnityEditor.Rendering.Universal
 {
+    using System;
+    using System.Linq;
+    using System.Linq.Expressions;
+    using System.Reflection;
     using UnityEngine;
-    using CED = CoreEditorDrawer<object>;
+    using UnityEngine.Rendering;
+    using UnityEngine.Rendering.Universal;
+    using CED = CoreEditorDrawer<UniversalRenderPipelineSerializedLight>;
 
     internal partial class UniversalRenderPipelineLightUI
     {
+        // should have the same int values as corresponding shape in LightType
+        enum AreaLightShape
+        {
+            None = 0,
+            Rectangle = 3,
+            Disc = 4
+        }
```

Comment:
Why do we not use `LightType `directly? All usages of this enum are reading values.

Reply:
It was code from trunk, changed :)